### PR TITLE
Use wit_bindgen in rust library

### PR DESCRIPTION
**Pull request by TiejunZhou-CN**

### CODE REVIEW

File: rust/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasi-nn"
-version = "0.6.0"
+version = "25.0.2"
```
Comment (abrown) [inline rust/Cargo.toml:3]: Can we just move this version to `0.7.0`?

Comment (TiejunMS) [reply rust/Cargo.toml]: Updated

Comment (abrown) [reply rust/Cargo.toml]: Thanks! I think @jlb6740 is going to take a look at this PR further; thanks for the contribution!

**Comment by jlb6740** [comment]
Hi @TiejunMS .. I think the workflow file causing the CI failure should be updated to omit regenerating bindings from witx. Did you want to submit that along with the changes here?

**Comment by TiejunMS** [comment]
> Hi @TiejunMS .. I think the workflow file causing the CI failure should be updated to omit regenerating bindings from witx. Did you want to submit that along with the changes here?

Sure. I can simply remove this workflow.

**Comment by TiejunMS** [comment]
@jlb6740 and @abrown , is there anything I can do to complete this PR? Thanks!

**Review by jlb6740 — APPROVED** [review]
Thanks .. looks good.

**Comment by TiejunZhou-CN** [comment]
Thanks @jlb6740 ! Do you know when this crate will be published?

**Pull request merged**